mmv1/products/firebase/AndroidApp.yaml

```diff
@@ -116,6 +132,13 @@ properties:
     description: |
       The SHA256 certificate hashes for the AndroidApp.
     item_type: Api::Type::String
+  - !ruby/object:Api::Type::String
```

Comment:
These KeyIds are referring to https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/apikeys_key  can we make them a resource-reference and point more precisely at the entity Devs should use?

Reply:
Last time we mentioned this, cross-service resource reference isn't supported yet. @melinath is it still the case?

Reply:
It should be safe to use ResourceRef if you want, but string is also fine. From another team member:

> functionally on the yaml side it just implements a validation to MMV1 build-time that the referenced field exists on the target resource if said resource exists in MMV1, but otherwise is just a string field that would be primarily used as reference to a field within another resource in your config. The main benefit of declaring a field to be one is it would allow more attributes to be accessed in custom code downstream, but if there is no custom code there is, again, effectively zero difference between a plain string field and a resource_reference.

Reply:
It turns out `google_apikeys_key` is DCL-based, so I guess a string it is.